src/screen/NearShops.js

```diff
@@ -0,0 +1,98 @@
+import React, {Component} from 'react';
+import {
+    FlatList,
+    Text,
+    StyleSheet,
+    View,
+    TouchableOpacity,
+    Image,
+} from 'react-native';
+import ShopRow from '../component/ShopRow';
+import * as Constants from '../common/constants';
+
+export default class NearShops extends Component {
+    static navigationOptions = {
+        title: 'VICINO A TE',
+    };
+    render() {
+        const {navigate} = this.props.navigation;
+
+        return (
+            <View style={styles.container}>
+                <FlatList
+                    data={[
```

Comment:
Please, use an external array with fixtures data. It will be easier to replace it with dynamic data